x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/models_list.tsx

```diff
@@ -151,10 +132,16 @@ export const ModelsList: FC = () => {
       const expandedItemsToRefresh = [];
 
       for (const model of response) {
-        const tableItem = {
+        const tableItem: ModelItem = {
           ...model,
+          // Extract model types
           ...(typeof model.inference_config === 'object'
```

Comment:
Do we need to check if this is not `null`?

Reply:
It can't be `null`. And it seems like now it's always an `object`. A while ago it was undefined for the build-in model `lang_ident_model_1 `, but now it contains `classification` config. 
So it's ok to keep as is